packages/server/lib/project-base.ts

```diff
@@ -399,7 +396,8 @@ export class ProjectBase<TServer extends Server> extends EE {
     return this.initSpecStore({ specs, config: updatedConfig })
   }
 
-  async initializePlugins (cfg, options) {
+  // TODO(tim): Improve this when we completely overhaul the rest of the code here,
```

Comment:
Yep we can nuke/rework most of this.  The reason it's still the way it is, is I was trying to keep it back compatible with the current desktop-gui to make reviewing/merging more easy, but I guess we can just abandon that consider how big a change we are introducing.